src/Pages/juheejang/Login/Login.js

```diff
@@ -1,11 +1,87 @@
-import React from "react";
+import React from 'react';
 import "./Login.scss";
-import { withRouter } from "react-router-dom";
+import { Link } from 'react-router-dom';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      idValue: '',
+      pwValue: '',
+      loginBtn: false
+    };
+  }
+
+  handleInput = (event) => {
+    const { name, value } = event.target;
+    this.setState({
+      [name]: value
+    },
+      this.checkLogin
+    )
+  }
+
+  checkLogin = () => {
+    const { idValue, pwValue } = this.state;
+    const isVal = idValue.includes("@") && pwValue.length >= 5;
+    this.setState({
+      loginBtn: isVal
+    })
```

Comment:
object에서 key,value 같으면 하나로 줄일 수 있습니다!
```suggestion
    const loginBtn = idValue.includes("@") && pwValue.length >= 5;
    this.setState({
      loginBtn,
    })
```

Reply:
저 변수선언한 loginBtn 값 자체가 true가 되기 때문에 바로 넣어줄 수 있나보네요! 확인 후 수정했습니다 !  😀